gr-filter/grc/filter_freq_xlating_fir_filter_xxx.block.yml

```diff
@@ -53,4 +53,13 @@ templates:
     - set_taps(${taps})
     - set_center_freq(${center_freq})
 
+cpp_templates:
+    includes: [ '#include <gnuradio/filter/freq_xlating_fir_filter.h>' ]
+    declarations: 'filter::freq_xlating_fir_filter::sptr ${id};'
```

Comment:
Is there a `_${type}` missing here in `declarations:`?

Reply:
Did it even work as it was before? Can you verify that it works now? We don't have a QA test, so we'll have to rely on people trying it out.